frontend/src/framework/store/store.package.ts

```diff
@@ -40,13 +42,14 @@ class Store {
                 auth: authReducer,
                 users: usersReducer,
             },
-            middleware: (getDefaultMiddleware) => {
-                return getDefaultMiddleware({
+            middleware: (getDefaultMiddleware) => [
+                ...getDefaultMiddleware({
                     thunk: {
                         extraArgument: this.extraArguments,
```

Comment:
we should add it to extra, in this case, we can call notification in the store

Reply:
can you clarify, please, how I can catch error in that case? who will be initiator of dispatch error action?